docs/_articles/en/faq.md

```diff
@@ -1,28 +1,32 @@
 ---
-title: 'FAQ: General'
+title: 'Frequently Asked Questions'
 lang: en
 ---
 
-# Frequently Asked Questions
-
 ## Table of Contents
+
```

Comment:
Are these newlines supposed to be actually rendered on the page? I'm not seeing them.